src/api.rs

```diff
@@ -7,6 +7,12 @@ use axum::{
 use crate::routes::{compile, create_gist, evaluate, static_css, static_html, static_js};
 use crate::GithubClient;
 use std::net::SocketAddr;
+use http::HeaderValue;
+use tower_http::cors::CorsLayer;
+
+let layer = CorsLayer::new().allow_origin(
```

Comment:
Looks like the linter wants this change:
```suggestion
const layer = CorsLayer::new().allow_origin(
```